sdk/cosmos/azure-cosmos-benchmark/src/test/java/com/azure/cosmos/benchmark/linkedin/data/TestInvitationDataGenerator.java

```diff
@@ -12,7 +12,7 @@
 
     @Test(groups="unit")
     public void testGenerate() {
```

Comment:
as code style we don't add "test" prefix to the our test methods.

Reply:
Fixed it in the latest revision